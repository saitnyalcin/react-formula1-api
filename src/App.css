.App {
  text-align: center;
}

.App-logo {
  height: 20vmin;
  pointer-events: none;
}
.footer-logo {
  margin: 1%;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-image: url('../src/images/schumacher-banner.jpg'); /* The image used */
  height: 500px; /* You must set a specified height */
  background-position: center; /* Center the image */
  background-repeat: no-repeat; /* Do not repeat the image */
  background-size: cover; /* Resize the background image to cover the entire container */

  min-height: 20vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

/* Flex box styling */
.flex-container {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;
  background-color: #282c34;
}

.flex-container > div {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  background-color: azure;
  width: 20em;
  margin: 1em;
  border-radius: 5%;
}

.flex-container h2 {
  margin: 2%;
}

.pilot-image img {
  margin: 1%;
  width: 12em;
  height: 12em;
  border-radius: 100%;
}

a {
  text-decoration: none;
}
/* unvisited link */
a:link {
  color: red;
}

/* visited link */
a:visited {
  color: red;
}

/* mouse over link */
a:hover {
  color: black;
}

/* selected link */
a:active {
  color: blue;
}
