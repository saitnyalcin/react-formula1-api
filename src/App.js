import React from "react";
import logo from "./images/tire.png";
import f1logo from "./images/F1_logo.png";
import "./App.css";
import "./style.scss";
import Formula from "./components/Formula";

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <div className="schumi_quote">
          Once something is a passion, the motivation is there...
        </div>
        <div>Season 2008</div>
      </header>

      <Formula />

      <div>
        <img src={f1logo} className="footer-logo" alt="logo" />
      </div>
    </div>
  );
}

export default App;
