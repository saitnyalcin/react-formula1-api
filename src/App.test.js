import React from "react";
import { shallow } from "enzyme";
import App from "./App";

describe("App component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  test("matches the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });

  test("renders the header of the Season 2008 page", () => {
    expect(wrapper.find("p").text()).toBe("Season 2008");
  });

  test("renders the nested Formula Component inside the App component", () => {
    expect(wrapper.find("Formula").exists()).toBe(true);
  });
});
