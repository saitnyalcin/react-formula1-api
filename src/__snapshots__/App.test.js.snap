// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches the snapshot 1`] = `ShallowWrapper {}`;

exports[`App component matches the snapshot: Season 2008 1`] = `ShallowWrapper {}`;
